### PR TITLE
Added objects with things to do

### PR DESCRIPTION
**Pull request by seanpmaloney**

### CODE REVIEW

**Review by matkatmusic — APPROVED** [review]

**Review by matkatmusic — APPROVED** [review]

**Review by matkatmusic — APPROVED** [review]

File: main.cpp [hunk 1]
```diff
@@ -12,76 +12,202 @@ What you create in this project will be used as the basis of Project 5 in the co
...
+5 properties:
+    1) highBandQ
+    2) midBand1Q
+    3) midband2Q
+    4) lowbandQ
+    5) highBand gain
+3 things it can do:
+    1) low shelf shape EQ
+    2) high shelf shape EQ
+    3) peak shape EQ
```
Comment (matkatmusic) [inline main.cpp]: These are objects, not actions
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -12,76 +12,202 @@ What you create in this project will be used as the basis of Project 5 in the co
...
+ */
+
+/*
+Thing 9) Aux Section
+5 properties:
+    1) aux send 1
+    2) aux send 2
+    3) aux send 3
+    4) aux send 1 level
+    5) aux send 2 level
```
Comment (matkatmusic) [inline main.cpp]: How does `aux send 1` differ from `aux send 1 level`?
How does `aux send 2` differ from `aux send 2 level`?
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 2 above)
Comment (seanpmaloney) [reply main.cpp]: You must first choose which aux to send to, then you select the level

Comment (seanpmaloney) [reply main.cpp]: so aux send 1 could be an int and level a float

File: main.cpp (hunk 1 above)
Comment (seanpmaloney) [reply main.cpp]: I was thinking more "create low shelf shaped EQ", but i changed them

**Review by matkatmusic — APPROVED** [review]

File: main.cpp [hunk 3]
```diff
@@ -13,75 +13,472 @@ What you create in this project will be used as the basis of Project 5 in the co
...
+    //Amount of profit each day
+    float dailyProfit = 2600.89f;
+
+    struct Kitchen
+    {
+        std::string specialOfTheDay;
+        std::string headChef;
+        bool isServingBreakfast;
+        int numWorkers;
+        bool isClean;
```
Comment (matkatmusic) [inline main.cpp]: How about some default values? 
- [x] revise

File: main.cpp [hunk 4]
```diff
@@ -13,75 +13,472 @@ What you create in this project will be used as the basis of Project 5 in the co
...
+    // Father's name
+    std::string dadName = "Joe";
+
+    struct Kid {
+        std::string kidName;
+        int age;
+        bool playsSports;
+        bool likesArt;
+        bool playsAnInstrument;
+    };
```
Comment (matkatmusic) [inline main.cpp]: Curly braces go on their own line
Give your member variables some default values
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]
some minor changes needed in your nested types.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**